store/store.go

```diff
@@ -2322,3 +2340,100 @@ func (s *Store) snapAction(ctx context.Context, currentSnaps []*CurrentSnap, act
 
 	return snaps, nil
 }
+
+var errUnexpectedConnCheckResponse = errors.New("unexpected response during connection check")
+
+var netDial = net.Dial
+
+func (s *Store) authConnCheck() ([]string, error) {
```

Comment:
@noise also asked not to do this, anyway alone without checking login it might not work anyway